src/w3c/templates/headers.js

```diff
@@ -253,6 +259,75 @@ export default (conf, options) => {
   </div>`;
 };
 
+function renderFeedback(conf) {
+  if (!conf.github) {
+    const hint = docLink`Use the ${"[github]"} configuration option to add a link to a repository.`;
+    showError("A link to a public repository is required.", name, {
+      hint,
+    });
+  }
+
+  if (!conf.github && !conf.wgPublicList) return;
+  const definitions = [];
+
+  // Github feedback...
+  if (conf.github) {
+    const issuesURL = new URL("./issues/", conf.github.repoURL);
+    definitions.push(
+      html`<dd><a href="${issuesURL}">${issuesURL.href}</a></dd>`
+    );
+  }
+
+  if (conf.wgPublicList) {
+    // The <a href="mailto:list?subject"> link for the public list
+    const mailToURL = new URL(`mailto:${conf.wgPublicList}@w3.org`);
+    const subject =
+      conf.subjectPrefix ?? `[${conf.shortName}] ${l10n.your_topic_here}`;
+    const mailingListLink = html`<a
+      href="${mailToURL.href}?subject=${encodeURIComponent(subject)}"
+      >${mailToURL.pathname}</a
+    >`;
+
+    // The subject line...
+    const subjectLine = conf.subjectPrefix
+      ? conf.subjectPrefix
+      : html`[${conf.shortName}] <em>${l10n.message_topic}</em>`;
```

Comment:
```suggestion
    const subjectLine = conf.subjectPrefix || html`[${conf.shortName}] <em>${l10n.message_topic}</em>`;
```

Reply:
Actually, here we probably want to do `conf.subjectPrefix ??`, then just allow `""`.